### PR TITLE
Add man docs for bundle install --force option

### PR DESCRIPTION
**Pull request by steverob**
Adding some documentation for the `--force` option of `install`. Based on issue #3625.

Could not think of more stuff to write about this option. Lemme know if this is enough. :) Thanks!

### CODE REVIEW

File: man/bundle-install.ronn [hunk 1]
```diff
@@ -79,7 +80,11 @@ update process below under [CONSERVATIVE UPDATING][].
   In [deployment mode][DEPLOYMENT MODE], Bundler will 'roll-out' the bundle for
   `production` use. Please check carefully if you want to have this option
   enabled in `development` or `test` environments.
-
+  
+* `--force`:
+  Force download every gem even if the required versions are already available
```
Comment (segiddins) [inline man/bundle-install.ronn]: `Force download every gem, even`

File: man/bundle-install.ronn [hunk 2]
```diff
@@ -79,7 +80,11 @@ update process below under [CONSERVATIVE UPDATING][].
   In [deployment mode][DEPLOYMENT MODE], Bundler will 'roll-out' the bundle for
   `production` use. Please check carefully if you want to have this option
   enabled in `development` or `test` environments.
-
+  
+* `--force`:
+  Force download every gem even if the required versions are already available 
+  in the system.
```
Comment (segiddins) [inline man/bundle-install.ronn]: `s/in the system/locally/`

**Comment by segiddins** [comment]
:+1:

**Comment by steverob** [comment]
@segiddins thanks!! On it. Also will fix the `malformed whitespace` problems.

**Comment by steverob** [comment]
@segiddins done. :+1:

**Pull request merged**